sites/partners/next.config.js

```diff
@@ -2,6 +2,7 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 
 const withTM = require("next-transpile-modules")([
```

Comment:
I'm wondering if we can change to the recommended way of doing this https://www.npmjs.com/package/next-transpile-modules
```
All features of next-transpile-modules are now natively built-in Next.js 13.1. Please use Next's transpilePackages option :)
```

But we don't necessarily have to do that as part of this PR

Reply:
https://github.com/bloom-housing/bloom/issues/3401